Doc/library/random.rst

```diff
@@ -572,14 +572,14 @@ Simulation of arrival times and service deliveries for a multiserver queue::
    including simulation, sampling, shuffling, and cross-validation.
 
    `Economics Simulation
-   <https://nbviewer.jupyter.org/url/norvig.com/ipython/Economics.ipynb>`_
+   <https://nbviewer.org/url/norvig.com/ipython/Economics.ipynb>`_
```

Comment:
@hugovk I believe that `nbviewer.jupyter.org` is the correct URL. Thanks. Same as the other link.

Reply:
When I click the `nbviewer.jupyter.org` links I'm redirected to `nbviewer.org`, is it different for you? We can keep the originals if you prefer, they do still work.

Reply:
https://nbviewer.jupyter.org/url/norvig.com/ipython/Economics.ipynb returns HTTP/2 302 and redirects to https://nbviewer.org/url/norvig.com/ipython/Economics.ipynb. While the current link works, the new link looks more canonical.

Reply:
Yeah, I get the redirect too. If I use only `nbviewer.org`, I was getting 500 errors but I suppose that may be due to Cloudflare's WARP on my machine. The URL that your updated seems to work in incognito mode so let's leave as you changed. Thanks!